testnet-contracts/scripts/mintTestTokens.js

```diff
@@ -21,33 +21,42 @@ module.exports = async () => {
   const NUM_ARGS = process.argv.length - 4;
 
   // Mint transaction parameters
-  const TOKEN_AMOUNT = 1000;
-
+  const TOKEN_AMOUNT = (1).toString().padEnd(20, "0")
+    console.log({TOKEN_AMOUNT})
   /*******************************************
    *** Command line argument error checking
    ***
    *** truffle exec lacks support for dynamic command line arguments:
    *** https://github.com/trufflesuite/truffle/issues/889#issuecomment-522581580
    ******************************************/
-  if (NETWORK_ROPSTEN) {
-    return console.error(
-      "Error: token minting on Ropsten network is not supported at this time"
-    );
-  } else {
-    if (NUM_ARGS !== 0) {
-      return console.error(
-        "Error: custom parameters for token minting are not supported at this time."
-      );
-    }
-  }
+  // if (NETWORK_ROPSTEN) {
```

Comment:
uncomment?

Reply:
They had token minting for ropsten turned off but I needed to do it when using xdai and think it is a useful feature. I left it as comment because i wasn't sure exactly why it was disabled for ropsten and would prefer to know more before removing it completely or adding it back. Maybe can ask @denalimarsh here?